src/api/forms/repositories/form-metadata-repository.js

```diff
@@ -18,7 +18,14 @@ export function list() {
     db.collection(METADATA_COLLECTION_NAME)
   )
 
-  return coll.find().limit(MAX_RESULTS).toArray()
+  return coll
+    .find()
+    .sort({
+      updatedAt: -1, // Primary sort: most recently updated documents first (-1 = descending order)
+      'updatedBy.displayName': 1 // Secondary sort: if update times are equal, sort by author name (1 = ascending order)
```

Comment:
Love this, exactly what we had to do on the prototype

Do we need tests for this sort of thing?

Reply:
Cheers @colinrotherham . I initially wrote tests, but thought there wasn't much point in the end as we're just mocking the results. We're limited by not having integration testing :/

Reply:
If we were mapping in the service layer, then 100% it should have coverage.

Reply:
Agreed this is out of scope of a unit test, but we should have integration tests to cover this. 

https://dev.azure.com/defragovuk/DEFRA-CDP/_backlogs/backlog/DXT%20Team/Features?workitem=266873 raised to cover that work.

Reply:
Agreed this is out of scope of a unit test, but we should have integration tests to cover this.

https://dev.azure.com/defragovuk/DEFRA-CDP/_backlogs/backlog/DXT%20Team/Features?workitem=266873 raised to cover that work.